app/frontend/src/components/form/Location.vue

```diff
@@ -123,12 +123,16 @@
               Details (eg:
               <em>"1km from HWY 1 at 100 mile house north on Logging Road"</em>)
             </label>
-            <v-text-field v-model="tentDetails" dense flat outlined solo />
+            <v-text-field
+              v-model="tentDetails"
+              :rules="[v => v.length <= 255 || 'Please enter 255 characters or less']"
+              dense flat outlined solo
+            />
           </v-col>
         </v-row>
       </div>
 
-      <v-checkbox v-model="accMotel" :readonly="reviewMode" label="Motel / Hotel in town"></v-checkbox>
+      <v-checkbox v-model="accMotel" :readonly="reviewMode" label="Worker’s Lodging Location (Motel, hotel, or other lodging)"></v-checkbox>
```

Comment:
can we change that apostrophe to ``` '``` (straight tick)?  maybe not an issue in Vue, but it is in word/pdf/cdogs, so don't want anyone cut/paste from here and mess up elsewhere.

Reply:
Done in the next commit

Reply:
Yes. that's a good idea. will do..